rust/operator-binary/src/superset_controller.rs

```diff
@@ -223,6 +216,8 @@ pub async fn reconcile_superset(superset: Arc<SupersetCluster>, ctx: Arc<Ctx>) -
         return Ok(Action::await_change());
     }
 
+    // Since rust 1.64 the '&*' became a clippy warning. It is needed though to dereference the Arc here
+    #[allow(clippy::explicit_auto_deref)]
```

Comment:
Try `superset.as_ref()` and remove the comment and clippy allow
```suggestion
```

Reply:
good suggestion, thanks! fixed: [08ff36f](https://github.com/stackabletech/superset-operator/pull/311/commits/08ff36fbc9ad0af03fdb27d48e12ef3a39ed58f6)